apps/08jquery-towers-of-hanoi/script.js

```diff
@@ -1,5 +1,25 @@
 'use strict';
 
 $(document).ready(function() {
-  // Put app logic here
-}
+  var $block = null;
+
+  function checkForWin(){
+    if($('div[data-stack="2"]').children().length === 4 || $('div[data-stack="3"]').children().length === 4) {
+      $('div#announce-game-won').text('You Won!');
+    }
+  }
+
+  $('[data-stack]').click(function() {
+    if(!$block) {
```

Comment:
Really enjoying how you wrote this code. You wrote out your checkForWin() function before you called it in your click listener (this will save you headaches later on.) Then you use the bang sign to simplify your 'if' statement. Bravo!